src/main/java/seedu/eylah/diettracker/logic/commands/AddCommand.java

```diff
@@ -0,0 +1,55 @@
+package seedu.eylah.diettracker.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.eylah.diettracker.logic.parser.CliSyntax.PREFIX_NAME;
+import static seedu.eylah.diettracker.logic.parser.CliSyntax.PREFIX_TAG;
+
+import seedu.eylah.diettracker.logic.commands.exceptions.CommandException;
+import seedu.eylah.diettracker.model.Model;
+import seedu.eylah.diettracker.model.food.Food;
+
+/**
+ * Adds a food to the food book.
+ */
+public class AddCommand extends Command {
+
+    public static final String COMMAND_WORD = "add";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a food to the food book. "
+            + "Parameters: "
+            + PREFIX_NAME + "NAME "
+            + PREFIX_TAG + "friends "
+            + PREFIX_TAG + "owesMoney";
```

Comment:
Do remember to remove this, I don't think it is relevant to the foodbook for the diet tracker.